tests/server.rs

```diff
@@ -298,3 +298,50 @@ fn expected_sat_time_is_rounded() {
     r".*<dt>timestamp</dt><dd><time>.* \d+:\d+:\d+ UTC</time> \(expected\)</dd>.*",
   );
 }
+
+#[test]
+fn server_runs_with_rpc_user_and_pass_as_env_vars() {
```

Comment:
Let's do the same style of test in index_runs_with_rpc_user_and_pass_as_env_vars

Reply:
see above